src/style/_layout.scss

```diff
@@ -844,9 +844,6 @@
         */
         .auth-wrapper 
         {
-            flex: none;
```

Comment:
Removing this will move the authorize button to the right on smaller screens when we have schemes. 

![test_after](https://github.com/swagger-api/swagger-ui/assets/44094416/a3172e7c-62d4-4fa6-96ff-788eba1d8f28)

Can you revert this change?

Reply:
done! I also updated the description, because the original problem no longer applies here.